data.sql

```diff
@@ -0,0 +1,6 @@
+/* Populate database with sample data. */
+
+INSERT INTO animals(date_of_birth, escape_attempts, neutered, weight_kg, name) VALUES ('02/03/2020', 0, true, 10.23, 'Agumon');
+INSERT INTO animals(date_of_birth, escape_attempts, neutered, weight_kg, name) VALUES ('15/11/2018', 2, true, 8, 'Gabumon');
+INSERT INTO animals(date_of_birth, escape_attempts, neutered, weight_kg, name) VALUES ('07/01/2021', 1, false, 15.04, 'Pikachu');
+INSERT INTO animals(date_of_birth, escape_attempts, neutered, weight_kg, name) VALUES ('12/05/2017', 5, true, 11, 'Devimon');
```

Comment:
I get an error when i run these queries `out of range date value ` i strongly suggest you stick to this format for date in SQL `YYYY-MM-DD` to avoid this type of errors 
![image](https://github.com/mirwaisfarahi/vet_clinic-database-project/assets/52497246/1b2fcc3f-d898-48fc-b134-4215a057bbc6)

Reply:
Thank you for the feedback. The date is changed to your recommended format.